### ISSUE TITLE
Cifar10 resnet18 

### ISSUE THREAD

**Issue by xinchiqiu** [negargoli/ReSprop#1]
Hi Negar, I’m currently working on sparsification methods and come across your inspiring ResProp paper. I’m trying to reproduce your Cifar10 on Resnet 18 results, but I’m unable to reach above 90% validation accuracy as you mentioned in your paper (Table 2). I’m using standard Resnet-18 architecture, batch size = 128, with SGD optimizer, 0.9 momentum, 0.1 learning rate, and reduce to 1/10 on 80 and 120 epoch ( as you mentioned in your paper). As I found that in your github you only shared your codes for the ImageNet, I’m wondering is it possible to share your codes for Cifar10 experiments?

Many thanks.
